src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -245,13 +258,6 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
 
-        if (session.protocolVersion.compareTo(ProtocolVersion.V4) < 0) {
-          bs.setToNull("message")
-          (1 to maxTagsPerEvent).foreach(tagId => bs.setToNull("tag" + tagId))
-        } else {
-          bs.unset("message")
-        }
```

Comment:
As stated in the README the library is only compatible with C* 2.1 versions if configured with  `cassandra-journal.cassandra-2x-compat=on`. Therefore everything related to tags should be skipped when `CassandraJournalConfig.enableEventsByTagQuery == false` (that property will be false when `cassandra-2x-compat=on`)